mysqld_exporter.go

```diff
@@ -262,6 +264,13 @@ const (
 		  FROM information_schema.schemata
 		  WHERE SCHEMA_NAME NOT IN ('mysql', 'performance_schema', 'information_schema')
 		`
+	queryResponseCheckQuery = `SELECT @@query_response_time_stats`
+	queryResponseTimeQuery = `
+		SELECT
+		    TIME, COUNT, TOTAL
+		  FROM information_schema.query_response_time
```

Comment:
Currently, I am getting:
```
# HELP mysql_info_schema_query_response_time_count The number of queries according to the length of time they took to execute.
# TYPE mysql_info_schema_query_response_time_count counter
mysql_info_schema_query_response_time_count{length="0.000001"} 102
mysql_info_schema_query_response_time_count{length="0.000010"} 146
mysql_info_schema_query_response_time_count{length="0.000100"} 2283
mysql_info_schema_query_response_time_count{length="0.001000"} 919
mysql_info_schema_query_response_time_count{length="0.010000"} 252
mysql_info_schema_query_response_time_count{length="0.100000"} 11
mysql_info_schema_query_response_time_count{length="1.000000"} 1
mysql_info_schema_query_response_time_count{length="10.000000"} 0
mysql_info_schema_query_response_time_count{length="100.000000"} 0
mysql_info_schema_query_response_time_count{length="1000.000000"} 0
mysql_info_schema_query_response_time_count{length="10000.000000"} 0
mysql_info_schema_query_response_time_count{length="100000.000000"} 0
mysql_info_schema_query_response_time_count{length="1000000.000000"} 0
# HELP mysql_info_schema_query_response_time_total Total time of queries according to the length of time they took to execute separately.
# TYPE mysql_info_schema_query_response_time_total counter
mysql_info_schema_query_response_time_total{length="0.000001"} 0
mysql_info_schema_query_response_time_total{length="0.000010"} 0.000654
mysql_info_schema_query_response_time_total{length="0.000100"} 0.088139
mysql_info_schema_query_response_time_total{length="0.001000"} 0.295455
mysql_info_schema_query_response_time_total{length="0.010000"} 0.481914
mysql_info_schema_query_response_time_total{length="0.100000"} 0.344209
mysql_info_schema_query_response_time_total{length="1.000000"} 0.267369
mysql_info_schema_query_response_time_total{length="10.000000"} 0
mysql_info_schema_query_response_time_total{length="100.000000"} 0
mysql_info_schema_query_response_time_total{length="1000.000000"} 0
mysql_info_schema_query_response_time_total{length="10000.000000"} 0
mysql_info_schema_query_response_time_total{length="100000.000000"} 0
mysql_info_schema_query_response_time_total{length="1000000.000000"} 0
```

Trying histogram I am getting:
```
# HELP mysql_info_schema_query_response_time Query response time distribution histogram.
# TYPE mysql_info_schema_query_response_time histogram
mysql_info_schema_query_response_time_bucket{length="0.000001",le="1e-06"} 139
mysql_info_schema_query_response_time_bucket{length="0.000001",le="+Inf"} 139
mysql_info_schema_query_response_time_sum{length="0.000001"} 0
mysql_info_schema_query_response_time_count{length="0.000001"} 139
...
<the same repeats for each table row>
```

Raw output:
```
mysql> SELECT TIME, COUNT, TOTAL FROM information_schema.query_response_time WHERE TIME <> 'TOO LONG';
+----------------+-------+----------------+
| TIME           | COUNT | TOTAL          |
+----------------+-------+----------------+
|       0.000001 |   124 |       0.000000 |
|       0.000010 |   179 |       0.000797 |
|       0.000100 |  2859 |       0.107321 |
|       0.001000 |  1085 |       0.335395 |
|       0.010000 |   269 |       0.522264 |
|       0.100000 |    11 |       0.344209 |
|       1.000000 |     1 |       0.267369 |
|      10.000000 |     0 |       0.000000 |
|     100.000000 |     0 |       0.000000 |
|    1000.000000 |     0 |       0.000000 |
|   10000.000000 |     0 |       0.000000 |
|  100000.000000 |     0 |       0.000000 |
| 1000000.000000 |     0 |       0.000000 |
+----------------+-------+----------------+
13 rows in set (0.00 sec)
```

Can you please explain the result I suppose to get with histogram? I don't see any existing examples in other exporters.